cmake/bob_robotics.cmake

```diff
@@ -256,12 +262,15 @@ macro(always_included_packages)
     if(NOT TARGET GLEW::GLEW)
         find_package(GLEW QUIET)
     endif()
+    if(NOT TARGET FreeImage::FreeImage)
```

Comment:
does gazebo include freeimage somehow?

Reply:
Yeah. Unfortunately it tries to create the ``FreeImage::FreeImage`` target unconditionally, so if you do ``find_package(gazebo)`` twice it fails because it already exists and errors out. Ugh.

Reply:
god I hate build systems